gcc/testsuite/gcc.target/riscv/cv-simd-subrotmj-div2-compile-1.c

```diff
@@ -0,0 +1,9 @@
+/* { dg-do compile } */
```

Comment:
I would be minded to merge all the subrotmj tests into one and have multiple scan-assembler lines at the end.  No need to change now, but would be a good cleanup later.